### DISCUSSION TITLE
Student Developer Pack

### DISCUSSION THREAD

**Discussion by kmou424** [community/community#66125] [GitHub Education]
### Hi everyone,

recently I have to request Student Developer Pack for study, but I received failed email from you.
I'm sure that I have fully provided all information needed.
Need help to re-verify my request.

**Comment by arafajs** [comment]
- Make sure that you are using a school-issued email address. If you don't have a school-issued email address, you can provide other documentation to prove your student status, such as a transcript or a letter from your school.
- Make sure that the information you provided in the application form matches the information in your GitHub profile. Your GitHub username, email address, and name should all match the information on your school records.
- Make sure that you are applying from a supported country or region. The GitHub Student Developer Pack is not available in all countries.
- If you have already applied for the Student Developer Pack in the past, and your application was rejected, you may need to wait 2 years before reapplying.

**Reply by kmou424** [reply]
I had provided the student id certificate with translated version, and my Github profile is matchs with my id card. But my request was refused by bot.
